frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -251,7 +254,8 @@ export function ChoroplethMap(props: ChoroplethMapProps) {
     const helperLegend = getHelperLegend(
       /* yOffset */ yOffsetNoDataLegend,
       /* xOffset */ xOffsetNoDataLegend,
-      /* overrideGrayMissingWithZeroYellow */ isCawp && !props.listExpanded
+      /* overrideGrayMissingWithZeroYellow */(isCawpOrHiv) &&
```

Comment:
can we use something like `containsZeros` instead of making it variable based? That'll allow us to expand further as more topics come in that might have zeros we want to have a unique color